lib/index.js

```diff
@@ -34,7 +34,7 @@ exports.plugin.dump = utils.toPromise(function (writableStream, opts, callback)
 
   var PouchDB = self.constructor;
 
-  var output = new PouchDB(self._db_name, {
+  var output = new PouchDB(self.name, {
```

Comment:
thanks, this is a new feature in pouchdb 6.0.0, so for backwards compat, should probably put an `||` here. I'll go ahead and do that though.